holopy/core/io/io.py

```diff
@@ -410,34 +411,82 @@ def load_average(filepath, refimg=None, spacing=None, medium_index=None, illum_w
         spacing = get_spacing(refimg)
 
     # read colour channels from refimg
+    channel_dict = {'0': 'red', '1': 'green', '2': 'blue'}
     if channel is None and refimg is not None and illumination in refimg.dims:
         channel = [i for i, col in enumerate(['red','green','blue']) if col in refimg[illumination].values]
```

Comment:
The enumerate here should be switched to iterating over `channel_dict.items()` since you've defined that dictionary. I don't think order matters here, and dictionaries are ordered as of python 3.6 anyway.

Reply:
Note this is a minor concern and I'm happy to go ahead and merge if you are sick of working on this and don't want to fix it.